moveit/scripts/create_readme_table.py

```diff
@@ -19,34 +20,31 @@ def create_header(ros_ubuntu_dict):
       header_lines = ' | '.join([header_lines, '-'*len(source), '-'*len(debian)])
   return '\n'.join([section_header, header, header_lines])
 
-def create_table_line_template(ros_ubuntu_dict):
-    ros_distros =  sorted(ros_ubuntu_dict.keys())
-    line = "{package_name}"
+def define_urls(target, params):
+    if target == 'src':
+        params['job'] = "{R}src_u{U}__{package}__ubuntu_{ubuntu}__source".format(**params)
+        params['url'] = "{base_url}/view/{R}src_u{U}/job/{job}".format(**params)
+    elif target == 'bin':
+        params['job'] = "{R}bin_u{U}64__{package}__ubuntu_{ubuntu}_amd64__binary".format(**params)
+        params['url'] = "{base_url}/view/{R}bin_u{U}64/job/{job}".format(**params)
+
+def create_line(package, ros_ubuntu_dict):
+    ros_distros = sorted(ros_ubuntu_dict.keys())
+    line = '\n' + package
```

Comment:
It seems a bit odd that the line starts with a newline, rather than ending with one, or leaving it up the the caller where to place the newline.

Reply:
That's the way you programmed it ;-)